tests/settings.py

```diff
@@ -120,6 +120,15 @@
     },
 ]
 
+OPENWISP_NOTIFICATION_TYPES = {
+    'dummy': {
+        'level': 'dummy level',
+        'verb': 'dummy verb',
+        'name': 'Dummy Notification Type',
+        'subject': '[openwisp-notification] Dummy Email Notification',
```

Comment:
I would remove `[openwisp-notification] `.

I think we should aim to have subjects like:

- `[{site.name}] {device.name} is not reachable`
- `[{site.name}] {device.name} is reachable`

We should strive to ensure that is possible. The variable site is from [django sites](https://docs.djangoproject.com/en/3.0/ref/contrib/sites/).